src/lib.rs

```diff
@@ -0,0 +1,7 @@
+extern crate libc;
+
+extern "C" {
```

Comment:
It turns out that `-l pgcommon` is passed when the lib is built -- not when the binary is built.